components/SelectControl.jsx

```diff
@@ -8,20 +9,19 @@ export default function SelectControl({ currentProjects }) {
   };
 
   return (
-    <div className="cmp-select-control">
-      <label>
-        <span className="cmp-select-control__label">Project</span>
-        <select
-          className="cmp-select-control__menu"
-          name="projects"
-          onChange={(e) => handleChange(e)}
-        >
-          <option className="cmp-select-control__option" value="all">All Projects</option>
-          {currentProjects.map((project) => (
-            <option key={project} className="cmp-select-control__option" value={project}>{project}</option>
-          ))}
-        </select>
-      </label>
-    </div>
+    <FieldsetGroup legend="Project">
+      <label htmlFor="project-select" className="cmp-select-control__label">Project</label>
```

Comment:
I've never used `htmlFor`, based on a quick search it looks like it's React's version of the `for` attribute, is that right?

Reply:
Yes it is! Just like `className` its one of those things you think React would be smart enough to be able to handle without the weird casing.